### PR TITLE
[DataStorage] Update Data Storage REST API description

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

In connection with the addition of the new REST API #373 made changes in the description `data_storage_api.yaml`

## Type of change

- [x] Documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Below is just an example and you can delete it when you contribute.

You can see result in [Swagger](https://editor.swagger.io)
 
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by t25kim — APPROVED** [review]
LGTM. Thanks!

File: api/data_storage_api.yaml [hunk 1]
```diff
@@ -277,6 +277,57 @@ paths:
                   $ref:  '#/components/examples/ResourceNotFound'
                 ConfigurationFileNotFound:
                   $ref:  '#/components/examples/ConfigurationFileNotFound'
+  '/api/v1/start/{start}/end/{end}/{ReadingCount}':
+    get:
+      tags:
+        - Data Storage
+      summary:  ""
+      description: "Multiple readings from a device at once based on start and end timestamp"
```
Comment (MoonkiHong) [inline api/data_storage_api.yaml:285]:
```suggestion
      description: "Multiple readings from a device at once based on start and end timestamps"
```

**Review by MoonkiHong — CHANGES_REQUESTED** [review]
@tdrozdovsky PTAL. 🙏

File: api/data_storage_api.yaml (hunk 1 above)
Comment (tdrozdovsky) [reply api/data_storage_api.yaml]: Done! Thank you!

**Review by MoonkiHong — APPROVED** [review]
+1

**Pull request merged**